frontend/src/App.vue

```diff
@@ -3,13 +3,14 @@
   import { RouterView } from 'vue-router'
   import Header from '@/components/header/Header.vue'
   import Footer from '@/components/footer/Footer.vue'
+  import Breadcrumb from '@/components/Breadcrumb.vue'
 
 </script>
 
 <template>
   <Header></Header>
-
   <main class="container pt-0 mt-2">
+    <Breadcrumb />
```

Comment:
Right now SnorCloud showing having issue with 'Breadcrum'.
Perhaps for now, maybe do <Breadcrumb></Breadcrumb> will solve it?

Reply:
This is a bug in SonarCloud dealing with Vue3, it doesn't think imports are used sometimes when they are. I discussed this with Derek. For now we are ignoring it as there is a SonarCloud fix for this issue, it just needs to be deployed to go into effect (no timeline given for this).